airbyte-integrations/bases/base-normalization/.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
can you just ignore build?

Reply:
sure. just posted in #dev but i thought what i had done here was our preferred best practice, but happy to just ignore `build`